test/acceptance/application-test.js

```diff
@@ -34,6 +34,11 @@ describe('Acceptance | application', function() {
         fixturesPath: 'tests'
       });
     }).then(function() {
+      app.editPackageJSON(function(pkg) {
+        pkg.devDependencies['ember-cli-fastboot'] = process.env.npm_package_devDependencies_ember_cli_fastboot;
```

Comment:
Why are you adding FastBoot here? Because `app.startServer` is still running `ember serve`, so where does FastBoot come into play? And where does `npm_package_devDependencies_ember_cli_fastboot` come from?